Patch release 1.4.1 changes (#302)
eclipse/che-che4z-explorer-for-endevor: CHANGELOG.md

You can find all notable changes to Explorer for Endevor in this document.

## [1.4.1] &ndash; 2023-01-26

### Fixed

> - Fixed an issue with the element fetching stuck in a loop after recovering from invalid credentials.

Reviewer: Will this suggesting work, too?
```suggestion
- Fixed an issue with recursive element fetching after recovering from invalid credentials.
```

Author: ```suggestion
- Fixed an issue with infinite element fetching after recovering from invalid credentials.
```
Just a small change, what do you think?

Reviewer: Then I'll add `loop` to that wording so that you have:
```suggestion
- Fixed an issue with an infinite element-fetching loop after recovering from invalid credentials.
```

Author: Awesome, I will take the last one 😉

### Changed

> - Element CCID is now optional element attribute because with certain Endevor configurations the CCID may not be specified at all.

Author: Yes, that is way better, thanks!
